app/views/index/normal.phtml

```diff
@@ -75,41 +75,77 @@ $today = @strtotime('today');
 
 			$this->renderHelper('index/normal/entry_header');
 
-		?><div class="flux_content" dir="auto">
+		?><article class="flux_content" dir="auto">
 			<div class="content <?= $content_width ?>">
-				<h1 class="title"><a target="_blank" rel="noreferrer" class="go_website" href="<?= $this->entry->link() ?>"><?= $this->entry->title() ?></a></h1>
-				<div class="subtitle">
-					<div class="website"><a href="<?= _url('index', 'index', 'get', 'f_' . $this->feed->id()) ?>" title="<?= _t('gen.action.filter') ?>: <?= $this->feed->name() ?>">
+				<header>
+					<div class="website"><a href="<?= _url('index', 'index', 'get', 'f_' . $this->feed->id()) ?>" title="<?= _t('gen.action.filter') ?>">
 						<?php if (FreshRSS_Context::$user_conf->show_favicons): ?>
-							<img class="favicon" src="<?= $this->feed->favicon() ?>" alt="✇" loading="lazy" />
-						<?php endif; ?>
-						<span><?= $this->feed->name() ?></span></a>
+							<img class="favicon" src="<?= $this->feed->favicon() ?>" alt="✇" loading="lazy" /><?php
+						endif; ?><span><?= $this->feed->name() ?></span></a>
 					</div>
-					<div class="date"><?= $this->entry->date() ?></div>
-					<?php
+					<div class="tags">
+						<?php
+						$tags = FreshRSS_Context::$user_conf->bottomline_tags ? $this->entry->tags() : null;
```

Comment:
Should probably use a new option instead, e.g.:

![image](https://user-images.githubusercontent.com/1008324/170956824-1d956334-d5a1-42fb-8bef-d551efbffde8.png)

Reply:
great idea.
It is now implemented